### DISCUSSION TITLE
What is the point of this tool?

### DISCUSSION THREAD

**Discussion by raggaerat** [UltimaHoarder/UltimaScraper#1601] [Q&A]
So with only fans you have to subscribe then buy content one by one to have access. This tool lets users download these images. However,  almost all conventional online video downloaders like tube offline and 9x can also do this. This script would be so much better if it allowed us to scrape all the content after subscribing. That would make it stand out. But right now unless you're a rich man who spend +1k on a model I don't see the point of using this tool. Regular folks only purchase like 10-20 vids anyway. Why not just use an offline tool like tube. 
Am I missing something here? 
It seems a perfect addition would be to allow us to get all the vids post subscription without having to individually spend 20+ per vid, no?
Is there a work around to getting the vids after you pay for the subscription without having to pay for the actual vids behind the paywall?
Tldr
(the tool requires you make an absurd amount of purchases before it becomes useful).

**Comment by UltimaHoarder** [comment]
Damn that's crazy. Anyway, close this issue.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by LunarPenguin** [comment]
No, it does not and will not download content you have not paid for. if it did OnlyFans would have a massive hole in their security and that would be fixed in a manner of **hours** and if this script bypassed OnlyFans' content protection it would be **DMCA'd** immediately after OF finds out the content protection was bypassed.

**Comment by kavinklain** [comment]
Then you've totally missed the point of this tool. This scraper is more for models that don't lock 98% their whole feed behind PPVs despite making you pay for the initial sub. There's plenty of great models that only need a sub and post everything publicly. This is meant for those types of models.
